flink/json-overview.yml

```diff
@@ -0,0 +1,46 @@
+type: json
+data:
+  url: /v1/overview
+  server: ${FLINK_HOSTPORT}
+
+observation:
+  - name: overview
+    metricNamespace: flink
+    path: $.
+
+    metric:
+      - name: slots.total
```

Comment:
Is it `jobs.slots.total`? Samo for other metrics below.

Reply:
Fixed for slots.total & slots.available